_adr/77/index.adoc

```diff
@@ -17,3 +17,58 @@ We have the choice:
 Homogeneous OSD model:: Each OSD cluster houses only Kafka instances with the same number of brokers
 Heterogeneous OSD model:: We place Kafka instances of different sizes in the same OSD cluster
 
+## Goals
+
+* Identify the preferred arrangement
+
+## Non-goals
+
+* Decisions about other considerations of running Kafka clusters with more than 3 brokers (e.g. cluster balancing, operational consequences etc).
+
+## Stakeholders
+* Eng
+* BU
+* SRE
+
+## Current Architecture
+
+In the current architecture we only support Kafka clusters of 3 brokers, so this is not a question that has needed to be addressed before.
+
+## Proposed Architecture
+
+We propose to adopt the heterogeneous model.
+
+The homogenous model would have a number of drawbacks:
+
+* It means scaling an existing Kafka instance up or down would become a problem of _Kafka cluster migration_.
+This is known to be a hard problem, which would therefore delay the provision of such a feature for an unknown period of time. In contrast, by using the heterogeneous model we can scale the Kafka clusters in place. 
+* It forces us to run additional OSD clusters immediately in each region, with the fixed costs that go with that. By following the heterogenous model we avoid the additional cost.
+
+### Threat Model
+
+* <Provide a link to the relevant threat model. You must either update an existing threat model(s) to cover the changes made by this ADR, or add a new threat model.
```

Comment:
should there be a ending '>' bracket as well?

Reply:
```suggestion
* No change to existing threat model
```